build/quote.go

```diff
@@ -27,19 +27,27 @@ import (
 
 // unesc maps single-letter chars following \ to their actual values.
 var unesc = [256]byte{
+	'a':  '\a',
+	'b':  '\b',
+	'f':  '\f',
 	'n':  '\n',
 	'r':  '\r',
 	't':  '\t',
+	'v':  '\v',
 	'\\': '\\',
 	'\'': '\'',
 	'"':  '"',
 }
 
 // esc maps escape-worthy bytes to the char that should follow \.
 var esc = [256]byte{
+	//'\a': 'a',
```

Comment:
Thanks, forgot to remove.

"\a" is the same as "\007", I'm not sure which should be the canonical form.

If a string is correctly escaped and contain either of these sequences it won't be changed by buildifier (e.g. the string `"\007\a"` will remain the same), but if there's an unknown string sequence (or single quotes are used instead of double quotes), the whole string will be unescaped and escaped again, e.g. `"\007\a\z"` will be replaced with either `"\a\a"` or `"\007\007"`. What do you think is better?

I currently made `\a` canonical (and the same for the other sequences) and updated the tests reflecting that, let me know if you think this is a bad decision.

Reply:
Sounds good, `\a` makes sense as the canonical format.